### ISSUE TITLE
ios Error: Error (Xcode): No profiles for 'xxx.xxx.xxx' were found

### ISSUE THREAD

**Issue by mannam95** [xeladu/flutter_build_pipeline#1]
I have created an ios azure pipeline based on the instructions mentioned in `pipeline.yml`.

I am getting the below error

```
Error (Xcode): No profiles for 'xxx.xxx.xxx' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'xxx.xxx.xxx'. 
Automatic signing is disabled and unable to generate a profile. To enable automatic signing, pass -allowProvisioningUpdates to xcodebuild.


It appears that there was a problem signing your application prior to installation on the device.
```

Can you share the file `ios/exportOptions.plist` mentioned in the `yml` file. How this file is created or is it auto-generated?

Can you please help me with this issue?

**Comment by xeladu** [comment]
[Here](https://medium.com/@marksiu/how-to-build-ios-project-with-command-82f20fda5ec5) is an article with more details about `exportOptions.plist` and an example that you can use. The file is required and created with XCode. Since it's project-specific, I can't provide a file that is always usable. Let me know if this helps.

**Comment by mannam95** [comment]
Unfortunately, it doesn't work for me. I think there is a problem with the `azure mac agent` or `mac os` related problem. 

For some reason, the provisioning profiles are not linked to the build on XCode and they fail during CI/CD.

I have tested on `codemagic` and everything works without any error.

**Comment by xeladu** [comment]
Yeah, unfortunately, there are more people having trouble with iOS builds than with Android. It's hard to tell what the solution is. Besides, I don't have a Mac so I can't dig into problems but I have this pipeline running successfully for one of my projects. So in theory, it should work.

If you ever find a solution, feel free to share it. Others might benefit from your efforts :) Good luck!

**Comment by abdu292** [comment]
@mannam95 I have the exact same issue. Just was wondering if you got a solution for this. Anything that you can share would be very much appreciated. Thanks in advance!

cc; @xeladu

**Comment by mannam95** [comment]
> [abdu292](/abdu292)

@abdu292 I have migrated to code-magic CI/CD, and it does everything excellent. You can give it a try!!

**Comment by abdu292** [comment]
Thanks @mannam95 for the response. I have a paid account on Azure DevOps and all of my other pipelines are on this. So thought to make this also as well under the same CI infra. However I'll give code magic a try for sure. Thanks again!

**Comment by abdu292** [comment]
Just thought to share in case if this helps someone. 

I found the working solution here.

https://github.com/hey24sheep/azure-flutter-tasks/issues/49#issuecomment-1309118882

Thanks!
